pom.xml

```diff
@@ -497,7 +497,7 @@
         <version.org.wildfly.arquillian>3.0.1.Final</version.org.wildfly.arquillian>
         <version.org.wildfly.core>18.0.0.Beta2</version.org.wildfly.core>
         <version.org.wildfly.extras.creaper>1.6.2</version.org.wildfly.extras.creaper>
-        <version.org.wildfly.http-client>1.1.8.Final</version.org.wildfly.http-client>
+        <version.org.wildfly.http-client>1.1.9.Final-SNAPSHOT</version.org.wildfly.http-client>
```

Comment:
Please remove the -SNAPSHOT suffix @chengfang

Reply:
the -SNAPSHOT suffix removed from `version.org.wildfly.http-client`